### ISSUE TITLE
Crashes while rebuilding Mappings in the database. 

### ISSUE THREAD

**Issue by sbstratos79** [funmaker/Hybooru#20]
I am getting this error after the latest update.
```
Database update detected, rebuilding...

Rebuilding Database!

Posts (33153/33153)             [####################] Done in 377ms
Urls (175457/175457)            [####################] Done in 623ms
Tags (77505/77505)              [####################] Done in 409ms
Mappings (633453/633453)        [####################] Done in 1.26s
Mappings (614400/610626)        [####################] 
/mnt/HDD/Home/Documents/Linux/git-clones/Hybooru/node_modules/pg-protocol/src/parser.ts:369
      name === 'notice' ? new NoticeMessage(length, messageValue) : new DatabaseError(messageValue, length, name)
                                                                    ^
error: duplicate key value violates unique constraint "mappings_pkey"
    at Parser.parseErrorMessage (/mnt/HDD/Home/Documents/Linux/git-clones/Hybooru/node_modules/pg-protocol/src/parser.ts:369:69)
    at Parser.handlePacket (/mnt/HDD/Home/Documents/Linux/git-clones/Hybooru/node_modules/pg-protocol/src/parser.ts:188:21)
    at Parser.parse (/mnt/HDD/Home/Documents/Linux/git-clones/Hybooru/node_modules/pg-protocol/src/parser.ts:103:30)
    at Socket.<anonymous> (/mnt/HDD/Home/Documents/Linux/git-clones/Hybooru/node_modules/pg-protocol/src/index.ts:7:48)
    at Socket.emit (node:events:513:28)
    at addChunk (node:internal/streams/readable:324:12)
    at readableAddChunk (node:internal/streams/readable:297:9)
    at Socket.Readable.push (node:internal/streams/readable:234:10)
    at TCP.onStreamRead (node:internal/stream_base_commons:190:23)
```
* I have tried removing the `hybooru` database and starting from scratch. 
* I have tried nodejs 18 lts and nodejs 19. 
* None of these steps helped.

**Comment by beam7894123** [comment]
I got the same problem so here what I think.
This happen because 
- both of your tag service have same tag in it
or
- you have more that one tag service

BUT here how I to fix it (for now)
- in `configs.json` 
- at `"tags"`
- edit `"services": null,` to `"services": ["your tag service go here"],`
but this come at cost of you can only use tag service that you specify
so not a long term solution

**Comment by sbstratos79** [comment]
> I got the same problem so here what I think. This happen because
> 
>     * both of your tag service have same tag in it
>       or
> 
>     * you have more that one tag service
> 
> 
> BUT here how I to fix it (for now)
> 
>     * in `configs.json`
> 
>     * at `"tags"`
> 
>     * edit `"services": null,` to `"services": ["your tag service go here"],`
>       but this come at cost of you can only use tag service that you specify
>       so not a long term solution

Thanks, this works. 
I'd much rather be able to use all my tag services. I'll keep the issue open for now.

**Comment by funmaker** [comment]
Fixed in [v1.7.0](https://github.com/funmaker/Hybooru/releases/tag/v1.7.0)
